cymetric/metrics.py

```diff
@@ -230,6 +230,39 @@ def agents(series):
 del _agentsdeps, _agentsschema
 
 
+# Transaction Quantity
+_transdeps = [
+    ('Materials', ('SimId', 'ResourceId', 'ObjId', 'TimeCreated', 'Units'), 
+        'Mass'),
+    ('Transactions', ('SimId', 'TransactionId', 'SenderId', 'ReceiverId', 
+        'ResourceId'), 'Commodity')
+    ]
+
+_transschema = [
+    ('SimId', ts.UUID), ('TransactionId', ts.INT), 
+    ('ResourceId', ts.INT), ('ObjId', ts.INT), 
+    ('TimeCreated', ts.INT), ('SenderId', ts.INT), 
+    ('ReceiverId', ts.INT), ('Commodity', ts.STRING), 
+    ('Units', ts.STRING), ('Mass', ts.DOUBLE)
+    ]
+
+@metric(name='TransactionQuantity', depends=_transdeps, schema=_transschema)
+def transaction_quantity(series):
+    """TransQuant metric returns the quantity of each transaction throughout 
+    the simulation.
+    """
+    trans_index = ['SimId', 'TransactionId', 'ResourceId', 'ObjId', 
+            'TimeCreated', 'SenderId', 'ReceiverId', series[1].name, 'Units']
```

Comment:
and so maybe `series[1].name` should be `Quantity` as well. I'll let you figure out the details.